jiffybox/cli.py

```diff
@@ -59,53 +56,66 @@ def jiffybox(api_key, verbose, output_format, pager):
     USE_PAGER = pager
 
 
-@jiffybox.group('box')
+@jiffybox.group('box', help='Manages boxes')
 def box():
     pass
 
 
-@box.command('list')
-@click.confirmation_option(prompt='This may take some time')
+@box.command('list', help='Lists all boxes in account')
 def list_boxes():
     print_data(API.boxes())
 
 
-@box.command('show')
+@box.command('show', help='Shows a single box')
 @click.argument('id')
 def show_box(id):
     box = API.box(id)
     print_data([box])
 
 
-@box.command('create')
+@box.command('create', help='Creates and start a new box')
 @click.argument('name')
 @click.argument('plan')
 @click.argument('distribution')
 @click.option('--sshkey/--no-sshkey', is_flag=True, default=True)
 def create_box(name, plan, distribution, sshkey):
-    box = API.create_box(name, plan, distribution=distribution,
+    box = API.create_box(name,
+                         plan,
+                         distribution=distribution,
                          use_sshkey=sshkey)
     print_data([box])
 
-@box.command('delete')
+
+@box.command('delete', help='Deletes a box')
 @click.argument('id')
 @click.option('--no-confirm', is_flag=True, default=False)
-def delete_box(id, no_confirm):
-    if no_confirm:
```

Comment:
I would like to keep the logic of the old `delete_box`. It is designed to avoid an unnecessary API rountrip and be a direct mapping of the library function.
We could introduce `wait-stop-and-delete` which uses the new logic. (Better names are welcome)